### ISSUE TITLE
Error in workorders (remedy): failed to combine water

### ISSUE THREAD

**Issue by rpherbig** [rpherbig/dr-scripts#750]
```
[workorders]>order 1
>
The attendant says, "You can purchase 10 splashes of water for 62 Kronars.  Just order it again and we'll see it done!"
>
[workorders]>order 1
The attendant takes some coins from you and hands you 10 splashes of water.
>
[workorders]>put my water in my backpack
You put your water in your backpack.
>
[workorders]>order 1
The attendant says, "You can purchase 10 splashes of water for 62 Kronars.  Just order it again and we'll see it done!"
>
[workorders]>order 1
The attendant takes some coins from you and hands you 10 splashes of water.
>
[workorders]>put my water in my backpack
You put your water in your backpack.
>
[workorders]>get water from backpack
You get some water from inside your backpack.
>
[workorders]>combine water
You must be holding both substances to combine them.  For more information, see HELP VERB COMBINE.
>
Barder puts his pestle in his backpack.
>
Barder gets an iron pestle from inside his backpack.
>
Barder roughly analyzes the blister cream.
>
[workorders: *** No match was found after 15 seconds, dumping info]
[workorders: messages seen length: 6]
[workorders: message: You must be holding both substances to combine them.  For more information, see HELP VERB COMBINE.]
[workorders: message: Barder puts his pestle in his backpack.]
[workorders: message: Barder gets an iron pestle from inside his backpack.]
[workorders: message:  Khri Dampen  (5 roisaen)]
[workorders: message: Barder roughly analyzes the blister cream.]
[workorders: message:  Khri Dampen  (5 roisaen)]
[workorders: checked against [/You combine/i]]
[workorders]>put my water in my backpack
You put your water in your backpack.
```

**Comment by Dartellum** [comment]
I have not seen or heard of a failure of this recently.

**Comment by rpherbig** [comment]
@Dartellum I just saw this today:
```
2016-09-21 06:56:07 -0400:--- Lich: go2 has exited.
2016-09-21 06:56:07 -0400:[workorders]>order 2
2016-09-21 06:56:07 -0400:The attendant says, "You can purchase a massive coal nugget for 212 Kronars.  Just order it again and we'll see it done!"
2016-09-21 06:56:07 -0400:[workorders]>order 2
2016-09-21 06:56:08 -0400:The attendant takes some coins from you and hands you a massive coal nugget.
2016-09-21 06:56:09 -0400:[workorders]>put my coal nugget in my backpack
2016-09-21 06:56:09 -0400:You put your nugget in your backpack.
2016-09-21 06:56:09 -0400:[workorders]>get water from my backpack
2016-09-21 06:56:09 -0400:You get some water from inside your backpack.
2016-09-21 06:56:09 -0400:[workorders]>count my water
2016-09-21 06:56:10 -0400:There are four parts left of the water.
2016-09-21 06:56:10 -0400:[workorders]>get water from backpack
2016-09-21 06:56:10 -0400:What were you referring to?
2016-09-21 06:56:10 -0400:[workorders]>put my water in my backpack
2016-09-21 06:56:10 -0400:You put your water in your backpack.
2016-09-21 06:56:10 -0400:[workorders]>get water from backpack
2016-09-21 06:56:10 -0400:You get some water from inside your backpack.
2016-09-21 06:56:10 -0400:[workorders]>combine water
2016-09-21 06:56:11 -0400:You must be holding both substances to combine them.  For more information, see HELP VERB COMBINE.
2016-09-21 06:56:25 -0400:[workorders: *** No match was found after 15 seconds, dumping info]
2016-09-21 06:56:25 -0400:[workorders: messages seen length: 3]
2016-09-21 06:56:25 -0400:[workorders: message: You must be holding both substances to combine them.  For more information, see HELP VERB COMBINE.]
2016-09-21 06:56:25 -0400:[workorders: message:  Khri Dampen  (8 roisaen)]
2016-09-21 06:56:25 -0400:[workorders: message:  Khri Dampen  (8 roisaen)]
2016-09-21 06:56:25 -0400:[workorders: checked against [/You combine/i]]
```

**Comment by Dartellum** [comment]
I believe I have this corrected in the the latest fix.

-- 
Regards,
Dr. Walter Wesley "Wes" Snyder V
484-300-4833 (h)
484-431-3355 (m)
484-302-7006 (f)

**Comment by rpherbig** [comment]
Should be fixed in #837
